Admin's Manual.md

```diff
@@ -121,9 +121,14 @@ discord:
     compact_mode: true # optional: disables avatars so messages take up less vertical space
 ```
 
-#### Using the Twitter integration
+#### Using the X integration
+Devzat supports sending updates about who is online on the X social media platform. However, because of the switch from free to paid API keys, This feature no longer is no longer updated.
 
-Devzat supports sending updates about who is online to Twitter. You need to make a new Twitter app through a [Twitter developer account](https://developer.twitter.com/en/apply/user)
+![API Key Pricing](https://global.discourse-cdn.com/twitter/original/3X/b/4/b42b6b7e4db2db3f51db70b439e9f111105db04d.png)
```

Comment:
We can let people find pricing themselves. Simply mentioning that the api is now paid and then linking to the new app page is enough